src/app/task.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
+import { TaskService } from './task.service';
```

Comment:
I would add the services to a dedicated directory.

Reply:
done!